src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -343,6 +394,39 @@ def includes_from_zcml(path):
     return modules, test_modules
 
 
+def includes_from_plone_fti(path):
+    modules = []
+    test_modules = []
+    for path, dirs, files in os.walk(path):
+        for fti in [os.path.abspath(os.path.join(path, filename))
+                    for filename in files
+                    if fnmatch.fnmatch(filename, '*.xml')]:
+            contents = open(fti).read()
+            # check that we are on a Dexterity FTI and
+            # not any other random XML file
+            if contents.find('meta_type="Dexterity FTI"') == -1:
+                continue
+
+            found = [module for module in
+                     re.findall(FTI_BEHAVIOR_PATTERN, contents)
+                     if not module.startswith('.') and module.find('.') != -1]
+
+            found += [module for module in
+                      re.findall(FTI_SCHEMA_PATTERN, contents)
+                      if not module.startswith('.')]
+            found += [module for module in
+                      re.findall(FTI_KLASS_PATTERN, contents)
+                      if not module.startswith('.')]
+            found += [module for module in
+                      re.findall(FTI_PERMISSION_PATTERN, contents)
```

Comment:
I basically skimmed over an FTI looked what seemed to be a dotted name and put it there, if you are sure that they are meaningless in regards of dependencies I will remove them, thanks for pointing it out!